### PR TITLE
Addition of Autojump Plugin

### PR DESCRIPTION
**Pull request by snuffop**
Add my $.02  with Autojump Plugin

### CODE REVIEW

File: plugins/README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Plugins extend the capabilities of `nnn`. They are _executable_ scripts (or bina
 
 | Plugin (a-z) | Description | Lang | Deps |
 | --- | --- | --- | --- |
+| autojump | AutoJump to dir/path | sh | [autojump](https://github.com/wting/autojump)
```
Comment (jarun) [inline plugins/README.md]: Please remove the link. `autojump` if fairly popular.

File: plugins/autojump [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env sh
+
+# Description: Autojump to directory using autojump
+# Source:      https://github.com/wting/autojump
```
Comment (jarun) [inline plugins/autojump]: Remove the link from here and add it to the "Requires" line.

File: plugins/autojump [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    AJ=autojump
+else
+    exit 1
+fi
+
+printf "Autojump to: "
+
+read -r DIR
+
+ODIR=$($AJ $DIR)
```
Comment (jarun) [inline plugins/autojump]: Please use small letters for variable names.

File: plugins/autojump [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+# Description: Autojump to directory using autojump
+# Source:      https://github.com/wting/autojump
+#
+# Requires: Autojump
+#
+# Shell: POSIX compliant
+# Author: Marty Buchaus
+
+if which autojump >/dev/null 2>&1; then
```
Comment (jarun) [inline plugins/autojump]: Please put the whole logic within this check.

File: plugins/autojump [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    AJ=autojump
```
Comment (jarun) [inline plugins/autojump]: That way `AJ` will be gone and you can use the command name directly.

File: plugins/autojump (hunk 3 above)
Comment (jarun) [inline plugins/autojump]: ODIR -> destdir

$DIR -> "$dir"

**Comment by snuffop** [comment]
Made all the Updates and pushed to my repo.. Do I need to re request.. Looks like the merge would correct things when complete?

File: plugins/autojump (hunk 3 above)
Comment (snuffop) [reply plugins/autojump]: Got this

File: plugins/autojump (hunk 5 above)
Comment (snuffop) [reply plugins/autojump]: Merged the Lines remove the var sub

File: plugins/autojump (hunk 3 above)
Comment (snuffop) [reply plugins/autojump]: Changed the Vars to lowercase

File: plugins/autojump (hunk 2 above)
Comment (snuffop) [reply plugins/autojump]: Moved the Link

File: plugins/README.md (hunk 1 above)
Comment (snuffop) [reply plugins/README.md]: Removed the link from Readme

File: plugins/autojump (hunk 4 above)
Comment (snuffop) [reply plugins/autojump]: done

File: plugins/autojump [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+# Description: Autojump to directory using autojump
+#
+# Requires: Autojump   https://github.com/wting/autojump
+#
+# Shell: POSIX compliant
+# Author: Marty Buchaus
+
+if which autojump >/dev/null 2>&1; then
+    printf "Autojump to: "
```
Comment (jarun) [inline plugins/autojump]: "Autojump to: " -> "jump to :"

File: plugins/autojump [hunk 7]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+
+# Description: Autojump to directory using autojump
+#
+# Requires: Autojump   https://github.com/wting/autojump
```
Comment (jarun) [inline plugins/autojump]: Requires: Autojump   https://github.com/wting/autojump

->

Requires: autojump - https://github.com/wting/autojump

File: plugins/autojump [hunk 8]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+
+# Description: Autojump to directory using autojump
```
Comment (jarun) [inline plugins/autojump]: Navigate to directory using autojump

File: plugins/README.md [hunk 9]
```diff
@@ -14,6 +14,7 @@ Plugins extend the capabilities of `nnn`. They are _executable_ scripts (or bina
 
 | Plugin (a-z) | Description | Lang | Deps |
 | --- | --- | --- | --- |
+| autojump | AutoJump to dir/path | sh | autojump |
```
Comment (jarun) [inline plugins/README.md]: Navigate to dir/path

File: plugins/autojump [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+#
+# Requires: Autojump   https://github.com/wting/autojump
+#
+# Shell: POSIX compliant
+# Author: Marty Buchaus
+
+if which autojump >/dev/null 2>&1; then
+    printf "Autojump to: "
+    read -r dir
+    odir=$(autojump $dir)
```
Comment (jarun) [inline plugins/autojump]: "$dir"

**Comment by jarun** [comment]
@snuffop just noticed you have closed this. What happened? We need to review the scripts carefully as they are used by many users.

**Comment by jarun** [comment]
Pushed at commit ef09f427209eab13c65d8abe986d67bd9b38a33d.

**Pull request closed without merging**